server/src/internalClusterTest/java/org/elasticsearch/health/node/ShardsCapacityHealthIndicatorServiceIT.java

```diff
@@ -54,6 +52,9 @@ public void testGreen() throws Exception {
         // index: 4 shards + 1 replica = 8 shards used (30 - 8 = 22 > 10 available shards)
         createIndex(4, 1);
 
+        ensureStableCluster(internalCluster.getNodeNames().length);
```

Comment:
I think is best if we add this to the method `fetchShardsCapacityIndicatorResult(..)` instead of having it duplicated along the 3 methods. WDYT?

Reply:
Good shout!